app/code/Magento/Wishlist/Block/Customer/Wishlist.php

```diff
@@ -63,6 +73,8 @@ public function __construct(
         $this->_helperPool = $helperPool;
         $this->currentCustomer = $currentCustomer;
         $this->postDataHelper = $postDataHelper;
+        $this->_scopeConfig = $context->getScopeConfig();
```

Comment:
_scopeConfig property is already present in \Magento\Framework\View\Element\AbstractBlock

Reply:
done